src/lib.rs

```diff
@@ -941,7 +964,204 @@ impl UInt {
         Self::new_saturating(self.0.saturating_pow(exp))
     }
 
-    // TODO: wrapping_* methods, overflowing_* methods
+    /// Wrapping (modular) addition. Computes `self + rhs`, wrapping around at the boundary of the
+    /// type.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::MAX.wrapping_add(UInt::from(0u32)), UInt::MAX);
+    /// assert_eq!(UInt::from(200u32).wrapping_add(UInt::MAX), UInt::from(199u32));
+    /// assert_eq!(UInt::MAX.wrapping_add(UInt::MAX), UInt::MAX - UInt::from(1u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_add(self, rhs: Self) -> Self {
+        Self::new_wrapping(match self.0.checked_add(rhs.0) {
+            Some(val) => val,
+            // Safety: The maximum value this addition can produce is `2 * UInt::MAX`,
+            // which comfortably fits in a u64.
+            None => unsafe { unreachable_unchecked() },
+        })
+    }
+
+    /// Wrapping (modular) subtraction. Computes self - rhs, wrapping around at the boundary of the type.
```

Comment:
Please check that the docs are rendering like the std docs.

```suggestion
    /// Wrapping (modular) subtraction. Computes `self - rhs`, wrapping around at the boundary of the type.
```